code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
```

Comment:
the same way we can go me coughs instead of *cough
or me faints. instead of *faint
or me gasps instead of *gasp
or...*continues on*